cpp/src/link_prediction/similarity_impl.cuh

```diff
@@ -36,13 +36,16 @@
 namespace cugraph {
 namespace detail {
 
+enum coefficient_t { JACCARD, SORENSEN, OVERLAP, COSINE };
```

Comment:
enum **class**

https://stackoverflow.com/questions/18335861/why-is-enum-class-considered-safer-to-use-than-plain-enum

Reply:
Changed.